pkg/sdn/api/v1/conversion.go

```diff
@@ -2,27 +2,75 @@ package v1
 
 import (
 	kapi "k8s.io/kubernetes/pkg/api"
+	conversion "k8s.io/kubernetes/pkg/conversion"
 
 	oapi "github.com/openshift/origin/pkg/api"
-	"github.com/openshift/origin/pkg/sdn/api"
+	newer "github.com/openshift/origin/pkg/sdn/api"
+)
+
+const (
+	KnownInvalidVNID = ^uint(0)
```

Comment:
@smarterclayton Can we change this to a `uint64` so that the value is completely predictable as opposed to "however many bits the system has"?